docs/synapse/userguides/syn_tools_feed.rst

```diff
@@ -118,114 +122,22 @@ From which we can issue Storm commands to interact with and validate the nodes t
-
-Ingest Example 2
-++++++++++++++++
-
-This example demonstrates loading a series of splices via the "syn.splices" format option. Splices are atomic edits made to the Cortex, so they are more granular, and thus more voluminous than just nodes. For instance, the storm command ``[it:host=1cad54991eaff5bba5d2015c29c3e3a3 :desc="synapse server" :name="syn007"]`` results in this set of splices (which have been saved to ``testsplices.yaml``).
-
-::
-
-  ---
-  - - node:add
-    - ndef:
-      - it:host
-      - 1cad54991eaff5bba5d2015c29c3e3a3
-      time: 1625087167677
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-  - - prop:set
-    - ndef:
-      - it:host
-      - 1cad54991eaff5bba5d2015c29c3e3a3
-      oldv: null
-      prop: .created
-      time: 1625087167677
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-      valu: 1625087167677
-  - - prop:set
-    - ndef:
-      - it:host
-      - 1cad54991eaff5bba5d2015c29c3e3a3
-      oldv: null
-      prop: desc
-      time: 1625087167679
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-      valu: synapse server
-  - - prop:set
-    - ndef:
-      - it:host
-      - 1cad54991eaff5bba5d2015c29c3e3a3
-      oldv: null
-      prop: name
-      time: 1625087167680
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-      valu: syn007
-  - - node:add
-    - ndef:
-      - it:hostname
-      - syn007
-      time: 1625087167680
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-  - - prop:set
-    - ndef:
-      - it:hostname
-      - syn007
-      oldv: null
-      prop: .created
-      time: 1625087167680
-      user: 267d945a32e3ae246ecf71e0bc6a620e
-      valu: 1625087167680
-  ...
-
-
-**Verifying the Data:**
-
-To load ``testsplices.yaml`` into a test Cortex to see the splices getting applied, we can run the ``feed`` tool like so:
-
-::
-
-  python -m synapse.tools.feed --test --debug --format "syn.splice" testsplices.yaml
-  
-Which drops us into a ``cmdr`` prompt, where we can verify that the ``it:host`` node and ``it:hostname`` nodes were created:
-
-::
-
-  cli> storm it:host
-  
-  it:host=1cad54991eaff5bba5d2015c29c3e3a3
-        .created = 2021/06/30 21:34:57.181
-        :desc = synapse server
-        :name = syn007
-        
-  complete. 1 nodes in 5 ms (200/sec).
-
-  cli> storm it:hostname
-  
-  it:hostname=syn007
-        .created = 2021/06/30 21:34:57.182
-  complete. 1 nodes in 5 ms (200/sec).
-
-
-**Loading the Data:**
-
-As before, once the data has been inspected and approved, we can point the ``feed`` tool at the Cortex we want to apply the splices to in order to apply them.
-
-::
-
-    python -m synapse.tools.feed --cortex tcp://cortex.vertex.link:4444/cortex00 --format 'syn.splice' 
-      testsplices.yaml
+With the ``chunksize`` parameter signifying that the ``feed`` tool should read two lines at a time from the file and
```

Comment:
`--chunksize 1` means read two lines at a time? Is that correct? Seems like `--chunksize 2` should mean two lines at a time!?

Reply:
There is some non-intuitive behavior here. Will clean those up separately.